### PR TITLE
cmd/snap/cmd_aliases: update stale aliases cmd help text

### PR DESCRIPTION
**Pull request by ZeyadYasser**
Remove section mentioning 'snap alias --reset' from aliases cmd which is now out of date as it was removed by commit e2d7e0ebd8d.

### CODE REVIEW

**Comment by MiguelPires** [comment]
Hey, thanks for opening this! I don't know if this is noted down elsewhere but we have a few labels that we use to make things easier on both the requester and the reviewers. One of these is the "Skip Spread" label, which skips the spread tests. We only use this when the changes are guaranteed not to have an effect on spread tests (like changes exclusive to unit tests or READMEs). This lowers the test load and also makes PRs much quicker to merge since spread tests take a long time and can sometimes be flaky. One thing to note is that the label has to be added before opening the PR so, if you add it after, just close and re-open the PR for the label to take effect. I added it to this PR since these changes are only on documentation and the aliases spread test doesn't cover them. Another label which signals to potential reviewer that the PR is small and easy to review is the "Simple" label. It doesn't have any functional behaviour, it's just so you don't have to wait a long time for something that could be reviewed quickly. There's also the "Needs Samuele review" label which signals that something can only be merged after an architecture review by Samuele (but that isn't the case is this PR). The "Needs security review" label signals that a +1 from the security team is required for merging. There are others as well but I think are the most common ones

**Comment by ZeyadYasser** [comment]
> If this is noted down elsewhere but we have a few labels that we use to make things easier on both the requester and the reviewers

@MiguelPires  Yes, it is mentioned in [CONTRIBUTING.md](https://github.com/snapcore/snapd/blob/b36a5c19217e08f68ce82c7076e4011914c1478c/CONTRIBUTING.md#labels), but I wasn't able to add labels, I thought I was missing permissions.

Thanks for the review!

**Comment by ZeyadYasser** [comment]
Yes, I have access now. Thank you!

**Review by mvo5 — APPROVED** [review]
This looks fine to me, thank you!

**Review by pedronis — APPROVED** [review]

**Pull request merged**
